features/bootstrap/ClapiContext.php

```diff
@@ -18,7 +18,6 @@ class ClapiContext extends CentreonContext
         "TRAP",
         "HTPL",
         "CG",
-        "LDAP",
```

Comment:
do not delete this and checkout `MON-12225-enhance-php-sql-failures-logging` on centreon-build from `develop`

Reply:
done